core/src/main/java/hudson/Functions.java

```diff
@@ -513,7 +513,15 @@ public static Map getEnvVars() {
     public static boolean isWindows() {
         return File.pathSeparatorChar==';';
     }
-    
+
+    public static boolean isMac() {
+        String osType = System.getenv("_system_name");
```

Comment:
Is there a reason you chose `_system_name` instead of the `os.name` described in the [Java properties](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html) tutorial?

Reply:
nope, other than i apparently missed that when i was looking for a reliable way to do this.